packages/fluentui/react-builder/src/components/ComponentTree.tsx

```diff
@@ -188,18 +190,20 @@ export const ComponentTree: React.FunctionComponent<ComponentTreeProps> = ({
 
   const selectedComponentId = selectedComponent?.uuid as string;
   const items: TreeItemProps[] =
-    tree.props?.children?.map(item =>
-      jsonTreeToTreeItems(
-        item,
-        selectedComponentId,
-        handleSelectComponent,
-        handleClone,
-        handleMove,
-        handleDeleteSelected,
-        handleAddComponent,
-        handleDeleteComponent,
-      ),
-    ) ?? [];
+    (Array.isArray(tree.props?.children) &&
+      tree.props?.children?.map(item =>
```

Comment:
Maybe it is a usage for `React.Children.map(children, item => ...)` ?

Reply:
I am not sure whether I am implementing it correctly
`React.Children.map(tree.props?.children, item => {
      return jsonTreeToTreeItems(
        item,
        selectedComponentId,
        handleSelectComponent,
        handleClone,
        handleMove,
        handleDeleteSelected,
        handleAddComponent,
        handleDeleteComponent,
      );
    })`

But this is not working. In console there is an error with "object is not a valid child.".